xcube/webapi/ows/stac/controllers.py

```diff
@@ -0,0 +1,373 @@
+                "title": "Information about the feature collections"
+            },
+            {
+                "rel": "search",
+                "href": f'{base_url}/catalog/search',
+                "type": "application/json",
+                "title": "Search across feature collections"
+            }
+        ],
+    }
+
+
+# noinspection PyUnusedLocal
+def get_conformance(ctx: DatasetsContext):
+    return {
+        "conformsTo": [
+            # TODO: fix this list
+            "http://www.opengis.net/spec/ogcapi-features-1/1.0/conf/core",
+            "http://www.opengis.net/spec/ogcapi-features-1/1.0/conf/oas30",
+            "http://www.opengis.net/spec/ogcapi-features-1/1.0/conf/html",
+            "http://www.opengis.net/spec/ogcapi-features-1/1.0/conf/geojson"
+        ]
+    }
+
+
+def get_collections(ctx: DatasetsContext, base_url: str):
+    return {
+        "collections": [
+            _get_datasets_collection(ctx, base_url)
+        ]
+    }
+
+
+def get_collection(ctx: DatasetsContext,
+                   base_url: str,
+                   collection_id: str):
+    _assert_valid_collection(ctx, collection_id)
+    return _get_datasets_collection(ctx, base_url, full=True)
+
+
+def get_collection_items(ctx: DatasetsContext,
+                         base_url: str,
+                         collection_id: str):
+    _assert_valid_collection(ctx, collection_id)
+    features = []
+    for dataset_config in ctx.get_dataset_configs():
+        dataset_id = dataset_config["Identifier"]
+        feature = _get_dataset_feature(ctx,
+                                       base_url,
+                                       dataset_id,
+                                       full=False)
+        features.append(feature)
+    return {
+        "type": "FeatureCollection",
+        "features": features,
+        "timeStamp": _utc_now(),
+        "numberMatched": len(features),
+        "numberReturned": len(features),
+    }
+
+
+def get_collection_item(ctx: DatasetsContext,
+                        base_url: str,
+                        collection_id: str,
+                        feature_id: str):
+    _assert_valid_collection(ctx, collection_id)
+    return _get_dataset_feature(ctx, base_url, feature_id, full=True)
+
+
+# noinspection PyUnusedLocal
+def search(ctx: DatasetsContext, base_url: str):
+    # TODO: implement me!
+    return {}
+
+
+# noinspection PyUnusedLocal
+def _get_datasets_collection(ctx: DatasetsContext,
+                             base_url: str,
+                             full: bool = False):
+    c_id, c_title, c_description = _get_collection_metadata(ctx.config)
+    return {
+        "stac_version": STAC_VERSION,
+        "stac_extensions": ["xcube"],
+        "id": c_id,
+        "title": c_title,
+        "description": c_description,
+        "license": "proprietary",
+        "keywords": [],
+        "providers": [],
+        "extent": {},
+        "summaries": {},
+        "links": [
```

Comment:
Shouldn't there be links here (with `'rel': 'item'`) to the actual datasets? With the current response I don't understand how I as an API user can actually obtain the list of available datasets.